src/test/scala/org/camunda/feel/impl/builtin/BuiltinNumberFunctionTest.scala

```diff
@@ -207,6 +207,12 @@ class BuiltinNumberFunctionsTest
     eval(" odd(2) ") should be(ValBoolean(false))
   }
 
+  "A odd() function" should "return true if negative number is odd" in {
```

Comment:
Here, we can use the `it` function to combine both cases under one root :)

```suggestion
  it should "return true if negative number is odd" in {
```

Reply:
done.